server/keyshare/keyshareserver/conf.go

```diff
@@ -63,6 +63,14 @@ type Configuration struct {
 	registrationEmailTemplates map[string]*template.Template
 
 	VerificationURL map[string]string `json:"verification_url" mapstructure:"verification_url"`
+	// number of tries allowed on pin before we start with exponential backoff
```

Comment:
A small nit pick: for all the other fields in this struct, the comment starts with a capital letter. If you're editing anyway, it's maybe nice to make this consistent by starting with a capital everywhere in this struct.